index1.html

```diff
@@ -0,0 +1,328 @@
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="card-button" id="thirdTitle">See Project</button>
+                </div> 
+                <div class="work-card">
+                    <div class="card-content flex-container ">
+                        <h4 class="title">4th: Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="card-button" id="fourthTitle">See Project</button>
+                </div> 
+                <div class="work-card">
+                    <div class="card-content  flex-container">
+                        <h4 class="title">5th: Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="card-button" id="fifthTitle">See Project</button>
+                </div> 
+              <div class="work-card">
+                    <div class="card-content flex-container">
+                        <h4 class="title">6th: Printing Data</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button" class="card-button" id="sixthTitle">See Project</button>
+                </div> 
+            </div>
+        </section><!-- #Recent Works -->
+        <!-- ======================
+            About me
+            ====================== -->
+
+            <section class="about-me" id="about-me">
+                <div class="about-text">
+                    <h2>About me</h2>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <button type="button">Get My Resume</button>
+                    <hr class="hr">
+                </div>
```

Comment:
The Figma template you are following has a background image, while your design is missing it. Please consider adding it.
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/170816720-7746b051-c7c9-49f0-94c3-c16ef6eb4238.png) | ![image](https://user-images.githubusercontent.com/33268927/170816734-af5b3595-f1a3-4901-b7de-1d4acad83ca0.png) |